spec/controllers/groups_controller_spec.rb

```diff
@@ -224,6 +224,13 @@
         user.reload
         expect(user.primary_group_id).to eq(nil)
       end
+
+      it "removes by user_email" do
+        xhr :delete, :remove_member, id: group.id, user_email: user.email
```

Comment:
I usually prefer that we assert that `expect(group.users.count).to eq(1)`. Otherwise if no users are created for some reason, the test will still pass